src/main/java/com/singlestore/fivetran/destination/JDBCUtil.java

```diff
@@ -400,4 +415,20 @@ static class TableNotExistException extends Exception {
             super("Table doesn't exist");
         }
     }
+
+    public static String getDatabaseName(SingleStoreConfiguration conf, String schema) {
+        if (conf.database() != null) {
+            return conf.database();
+        } else {
+            return schema;
+        }
+    }
+
+    public static String getTableName(SingleStoreConfiguration conf, String schema, String table) {
+        if (conf.database() != null) {
+            return String.format("%s_%s", schema, table);
```

Comment:
I'd use two underscores here to make it easier to distinguish between prefix (schema) and a regular table name like `my_dev_table`

Reply:
Good point. Changed this.